ballerina/tests/configs_negative_tests.bal

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org)
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/test;
```

Comment:
```suggestion

import ballerina/test;
```

Reply:
This is the default behaviour of the formatter. @poorna2152 can you please confirm if that's the expected behaviour?

Reply:
Formatter atm removes empty lines between the license header and the imports. There is an [issue](https://github.com/ballerina-platform/ballerina-lang/issues/41613) to track this and a open [pr](https://github.com/ballerina-platform/ballerina-lang/pull/41481) fixing this.

Reply:
@poorna2152 ack, thanks for the info.

@Nuvindu in that case shall we proceed with the default formatting behaviour ftm (otherwise there will be inconsistencies in between the source files) and, this will be automatically fixed with the upcoming releases.

Reply:
Yeah, sure